### PR TITLE
[Blog] Kubecon EU 2023

### PR DESCRIPTION
**Pull request by svrnm**
Announcements around KubeCon EU for the blog, I reused some of the structure and wording from the Kubecon NA announcement. Please review

Preview: https://deploy-preview-2534--opentelemetry.netlify.app/blog/2023/kubecon-eu/

### CODE REVIEW

File: content/en/blog/2023/kubecon-eu.md
Comment (frzifus) [inline content/en/blog/2023/kubecon-eu.md:55]: oh nice :D

**Review by frzifus — APPROVED** [review]
thx @svrnm

File: content/en/blog/2023/kubecon-eu.md
Comment (pavolloffay) [inline content/en/blog/2023/kubecon-eu.md:42]: There will be as well Jaeger project booth, do you want to list it here?

Comment (svrnm) [reply content/en/blog/2023/kubecon-eu.md:42]: Let me think about that, because if we list jaeger, we might also consider to list other observability projects (prometheus, pixie, ...)

**Review by mtwo — APPROVED** [review]

File: content/en/blog/2023/kubecon-eu.md
Comment (arminru) [inline content/en/blog/2023/kubecon-eu.md:50]: I think it would be worth pointing out that for the project meeting, one does _not_ need an all-access pass (right?) but for the Observability Day they would and those are already sold out as far as I know.
The ticket structure was recently changed which has led to some confusion already.

Comment (arminru) [reply content/en/blog/2023/kubecon-eu.md:50]:
```suggestion
The [OpenTelemetry project meeting](https://sched.co/1JWS7) is to meet and
network with the OpenTelemetry maintainers and core contributors. This meeting
will be held 13:00 - 15:00 on Tuesday April 18, 2023.
_This meeting can be attended with the standard in-person pass and does not require an "all-access" pass._

[Observability Day][] _fosters collaboration, discussion, and knowledge sharing
of cloud-native observability projects_. This event will be held on April 18,
2023 from 9:00 - 17:00. There will be several sessions on OpenTelemetry as well.
_This co-located event is not included in the general in-person pass but only in the "in-person all-access" pass. See the [Kubecon website](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/register/) for more details._
```
^ at least that's my understanding

File: content/en/blog/2023/kubecon-eu.md
Comment (alolita) [reply content/en/blog/2023/kubecon-eu.md:55]: @svrnm Thanks for putting this post together. Can you add an image or two from Kubecon NA as well as for OTel swag :-)

**Review by alolita — APPROVED** [review]
lgtm

File: content/en/blog/2023/kubecon-eu.md
Comment (svrnm) [reply content/en/blog/2023/kubecon-eu.md]: You're right @arminru! That's a good call out

File: content/en/blog/2023/kubecon-eu.md
Comment (svrnm) [reply content/en/blog/2023/kubecon-eu.md:57]: @alolita happy to do so, but I don't have any images, so if you have some, please send them over and I can add them in :-)

**Review by arminru — APPROVED** [review]

File: content/en/blog/2023/kubecon-eu.md
Comment (djaglowski) [inline content/en/blog/2023/kubecon-eu.md:40]:
```suggestion
- **[Connected Observability Pipelines in the OpenTelemetry Collector](https://sched.co/1HyXb)**<br>
  by Daniel Jaglowski, observIQ<br> Friday, April 21 • 11:55 - 12:30
- **[Tutorial: Exploring the Power of OpenTelemetry on Kubernetes](https://sched.co/1HyZ3)**<br>
  by Pavol Loffay, Benedikt Bongartz & Yuri Oliveira Sa, Red Hat, Severin
  Neumann, Cisco & Kristina Pathak, LightStep<br> Friday, April 21 • 16:00 -
  17:30
```

**Review by jpkrohling — APPROVED** [review]

File: content/en/blog/2023/kubecon-eu.md
Comment (arminru) [reply content/en/blog/2023/kubecon-eu.md:42]: Nice! 👏

**Review by djaglowski — APPROVED** [review]

File: content/en/blog/2023/kubecon-eu.md
Comment (chalin) [inline content/en/blog/2023/kubecon-eu.md:13]:
```suggestion
The OpenTelemetry project maintainers, members of the governance committee,
and technical committee are excited to be at [KubeCon EU][] in Amsterdam from
April 18 - 21, 2023!
```

File: content/en/blog/2023/kubecon-eu.md
Comment (chalin) [inline content/en/blog/2023/kubecon-eu.md:17]:
```suggestion
Come meet us at the [project booth][], listen to our talks or learn with us
during workshops. We are looking forward to hear from you, in particular how you are using
OpenTelemetry or contributing to the project.
```

File: content/en/blog/2023/kubecon-eu.md
Comment (chalin) [inline content/en/blog/2023/kubecon-eu.md:20]:
```suggestion
You will find below a list of talks and other activities related to
OpenTelemetry.
```

**Review by chalin — COMMENTED** [review]
Very nice. See inline copyedits (a few more to follow).

File: content/en/blog/2023/kubecon-eu.md
Comment (mhausenblas) [inline content/en/blog/2023/kubecon-eu.md:15]: As an end-user reading this, I immediately wonder: when and where can I meet folks at the booth? Is that every day all day? Where?

File: content/en/blog/2023/kubecon-eu.md
Comment (mhausenblas) [inline content/en/blog/2023/kubecon-eu.md:22]: Add EMEA end-user session? CC: @martinjt

**Review by mhausenblas — CHANGES_REQUESTED** [review]
LGTM, left a view comments and also let's make sure this time around there are OpenTelemetry stickers ;)

File: content/en/blog/2023/kubecon-eu.md
Comment (svrnm) [reply content/en/blog/2023/kubecon-eu.md:22]: yes, please, gimme the times & dates and I add them

Comment (mhausenblas) [reply content/en/blog/2023/kubecon-eu.md:22]: So, we haven't fixed that yet (unless I'm missing something), would it make sense to do it during the pavilion hours?

File: content/en/blog/2023/kubecon-eu.md
Comment (chalin) [inline content/en/blog/2023/kubecon-eu.md:56]:
```suggestion

> <i class="far fa-exclamation-triangle"></i> **IMPORTANT access note**: You
> need an _in-person all-access_ pass for access to **Observability Day**. For
> details, see [KubeCon
> registration](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/register/).
```

File: content/en/blog/2023/kubecon-eu.md
Comment (chalin) [inline content/en/blog/2023/kubecon-eu.md:65]:
```suggestion
You will find us in the Solutions Showcase in Hall 5, Kiosk Number 20. [Pavilion hours][] are as follows:
```

You'll also need the corresponding change to the mardown-link definition made below.

File: content/en/blog/2023/kubecon-eu.md
Comment (chalin) [inline content/en/blog/2023/kubecon-eu.md:76]:
```suggestion
Come join us to listen, learn, and get involved in OpenTelemetry.
```

**Comment by avillela** [comment]
@svrnm - Austin, @reese-lee, @musingvirtual, and I will be speaking at the Maintainer's Track on Wednesday, April 20th, 11:00 - 12:00

FYI - don't see it on the schedule yet, and we don't have a proper title for the talk, but I want to put this as a placeholder so we don't forget about it.

File: content/en/blog/2023/kubecon-eu.md
Comment (svrnm) [reply content/en/blog/2023/kubecon-eu.md]: I changed the intro to _Read on to learn about all the things related OpenTelemetry during KubeCon._ and leave it to the rest of the content to guide people to the right dates and locations, wdyt?

File: content/en/blog/2023/kubecon-eu.md
Comment (svrnm) [reply content/en/blog/2023/kubecon-eu.md:18]: I would guess so, probably worth checking at #otel-maintainers slack.

**Comment by svrnm** [comment]
It would be great to push out this blog post coming Monday, I set the date accordingly.

@avillela @mhausenblas if you don't have fixed dates yet we always can update the blog post later.

File: content/en/blog/2023/kubecon-eu.md
Comment (austinlparker) [inline content/en/blog/2023/kubecon-eu.md:50]: Please add that observability day will also be live-streamed if you have a virtual ticket

Comment (svrnm) [reply content/en/blog/2023/kubecon-eu.md:50]: fixed

**Comment by svrnm** [comment]
> @svrnm Can you please add a link to the community survey? https://docs.google.com/forms/d/e/1FAIpQLSdKm6oLYRXlZOhEZMVmjoIn4eBToVYNmF6fwpm5GAIipQmPxA/viewform?pli=1
> 
> If you need accompanying text, maybe something like this works:
> 
> > Help us improve the project by sharing your thoughts and feedback about your OpenTelemetry adoption, implementation, and usage! We will create action items from your comments as appropriate. Check [#otel-user-research](https://cloud-native.slack.com/archives/C01RT3MSWGZ) in CNCF's Slack instance for survey results and action item updates, to come after KubeCon EU.

added

File: content/en/blog/2023/kubecon-eu.md
Comment (svrnm) [reply content/en/blog/2023/kubecon-eu.md:40]: not really sure how to incorporate that best, if you have any suggestions let me know, otherwise I will skip it.

File: content/en/blog/2023/kubecon-eu.md
Comment (svrnm) [reply content/en/blog/2023/kubecon-eu.md:59]: I'll leave it like it is for now, if anyone has something to add, we can do this through a follow up PR

**Review by austinlparker — APPROVED** [review]

**Review by cartermp — APPROVED** [review]
Let's see if we can make this the most approved PR in the history of the repo

**Pull request merged**
